backend/src/main/java/com/sdacademy/budgettracker/service/MotivatingMessageService.java

```diff
@@ -0,0 +1,25 @@
+package com.sdacademy.budgettracker.service;
+
+import com.sdacademy.budgettracker.entity.MotivatingMessage;
+import com.sdacademy.budgettracker.repository.MotivatingMessageRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Random;
+
+@Service
+public class MotivatingMessageService {
+
+    @Autowired
+    private MotivatingMessageRepository repository;
+
+    public MotivatingMessage getRandomMessage() {
+        List<MotivatingMessage> messages = repository.findAll();
```

Comment:
Not efficient approach to pull all messages from the database, to choose and return only one of them. Doesn't make sense. Try to find a way, how to pull only one random row from db.